pkgs/development/tools/build-managers/gprbuild/gpr-project-darwin-rpath-hook.sh

```diff
@@ -0,0 +1,10 @@
+fixDarwinRpath() {
+    for f in $(find $out -type f -executable); do
+        install_name_tool -id $f $f || true
+        for rpath in $(otool -L $f | grep rpath | awk '{print $1}'); do
```

Comment:
Not very familiar with darwin, but wouldn't it be safer to grep for `@rpath`? (Or even `^@rpath` depending on were in the line it'd appear?)